.gitignore

```diff
@@ -10,4 +10,6 @@ CMakeCache.txt
 **/CMakeFiles/*
 .vimrc
 gymfc.json
-env/
+.idea
+venv
```

Comment:
remove this to stay consistent with the README that creates an environment named env